### ISSUE TITLE
tensorflow.python.framework.errors_impl.UnimplementedError:  Fused conv implementation does not support grouped convolutions for now.

### ISSUE THREAD

**Issue by eapache** [jhuus/HawkEars-TensorFlow#1]
Ubuntu 20.04, no CUDA.

Tried analyzing a file from Hamilton Boiacoustics (Specifically [this one](https://archive.org/download/HNCAM01520210514/HNCAM015_20210514_143225.mp3)).

```
Analyzing /home/eapache/Downloads/HNCAM015_20210514_143225.mp3
Traceback (most recent call last):
  File "analyze.py", line 186, in <module>
    analyzer.run()
  File "analyze.py", line 171, in run
    self._analyze_file(file_path)
  File "analyze.py", line 115, in _analyze_file
    specs = self._get_specs(offsets)
  File "analyze.py", line 82, in _get_specs
    spec_list = self.audio.get_spectrograms(offsets)
  File "/home/eapache/src/github.com/jhuus/HawkEars/core/audio.py", line 244, in get_spectrograms
    is_noise, high_max = self.binary_classifier.check_for_noise(specs, low_idx, high_idx, low_mult)
  File "/home/eapache/src/github.com/jhuus/HawkEars/core/binary_classifier.py", line 64, in check_for_noise
    predictions = self.model.predict(self.specs)
  File "/home/eapache/.local/lib/python3.8/site-packages/tensorflow/python/keras/engine/training.py", line 1727, in predict
    tmp_batch_outputs = self.predict_function(iterator)
  File "/home/eapache/.local/lib/python3.8/site-packages/tensorflow/python/eager/def_function.py", line 889, in __call__
    result = self._call(*args, **kwds)
  File "/home/eapache/.local/lib/python3.8/site-packages/tensorflow/python/eager/def_function.py", line 956, in _call
    return self._concrete_stateful_fn._call_flat(
  File "/home/eapache/.local/lib/python3.8/site-packages/tensorflow/python/eager/function.py", line 1960, in _call_flat
    return self._build_call_outputs(self._inference_function.call(
  File "/home/eapache/.local/lib/python3.8/site-packages/tensorflow/python/eager/function.py", line 591, in call
    outputs = execute.execute(
  File "/home/eapache/.local/lib/python3.8/site-packages/tensorflow/python/eager/execute.py", line 59, in quick_execute
    tensors = pywrap_tfe.TFE_Py_Execute(ctx._handle, device_name, op_name,
tensorflow.python.framework.errors_impl.UnimplementedError:  Fused conv implementation does not support grouped convolutions for now.
	 [[node model/batch_normalization_4/FusedBatchNormV3 (defined at /home/eapache/src/github.com/jhuus/HawkEars/core/binary_classifier.py:64) ]] [Op:__inference_predict_function_26140]

Function call stack:
predict_function
```

**Comment by eapache** [comment]
FWIW when installing dependencies I noticed this "error" which did not prevent it from continuing or succeeding:
`ERROR: imageio 2.13.3 has requirement pillow>=8.3.2, but you'll have pillow 7.0.0 which is incompatible.`

**Comment by jhuus** [comment]
This appears to be fixed in Tensorflow 2.7.0 (https://github.com/tensorflow/tensorflow/commit/7b8db6083b34520688dbc71f341f7aeaf156bf17). However, I tried 2.7.0 and got the same error, so for now we'll have to say that CUDA is required.

**Comment by jhuus** [comment]
Fixed in latest commit, which changed the model to avoid using Conv2D(groups=...).
